assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -193,15 +239,18 @@ function MultiSelectionMovable({ selectedElements }) {
           setTransformStyle(element.id, target, sFrame);
         });
       }}
-      onDragGroupStart={({ events }) => {
+      onDragGroupStart={({ events, inputEvent }) => {
+        startEventTracking(inputEvent);
```

Comment:
Note: The `onDragGroupStart` and `onDragGroupEnd` is basically the `mouseDown` and `mouseUp` event handling of the multi-selection.